sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/LiteralsShouldNotBePassedAsLocalizedParameters.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.ComponentModel;
+
+namespace Tests.Diagnostics
+{
+    class Program

```

Comment:
We report if the value is constant. If it is not - we don't care, hence a test with ResourceManager is pointless. I will add tests with field and local const.